pkg/kv/kvserver/raftlog/entry.go

```diff
@@ -28,41 +28,41 @@ import (
 	"go.etcd.io/raft/v3/raftpb"
 )
 
-// EncodingVersion determines the RaftCommandEncodingVersion for a given
+// EncodingVersion determines the EntryEncoding for a given
 // Entry.
-func EncodingVersion(ent raftpb.Entry) (kvserverbase.RaftCommandEncodingVersion, error) {
+func EncodingVersion(ent raftpb.Entry) (EntryEncoding, error) {
```

Comment:
This one still mentions "version". How about `raftlog.EncodingOf(ent)`?

Reply:
👍🏽